fetch_ikfast_plugin/update_ikfast_plugin.sh

```diff
@@ -1 +1 @@
-rosrun moveit_kinematics create_ikfast_moveit_plugin.py --search_mode=OPTIMIZE_MAX_JOINT --srdf_filename=fetch.srdf --robot_name_in_srdf=fetch --moveit_config_pkg=fetch_moveit_config fetch arm fetch_ikfast_plugin torso_lift_link gripper_link /home/alex/ros/melodic/src/fetch_ros/fetch_ikfast_plugin/src/fetch_arm_ikfast_solver.cpp
\ No newline at end of file
+rosrun moveit_kinematics create_ikfast_moveit_plugin.py --search_mode=OPTIMIZE_MAX_JOINT --srdf_filename=fetch.srdf --robot_name_in_srdf=fetch --moveit_config_pkg=fetch_moveit_config fetch arm fetch_ikfast_plugin torso_lift_link wrist_roll_link /home/ijwo/ros/melodic/fetch-sim-test/src/fetch_ros/fetch_ikfast_plugin/src/fetch_arm_ikfast_solver.cpp
```

Comment:
:+1: my mistake, I used `gripper_link` instead of `wrist_roll_link` thanks for fixing it.